Aspire Budgeting/Facilities/StateManager.swift

```diff
@@ -133,10 +153,14 @@ final class StateManager: ObservableObject {
     }
   }
 
-  func transition(to nextState: State) {
+  func hasDefaultFile() {
+    transition(to: .hasDefaultFile)
+  }
+  
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/aspirebudgetingmobile/aspirebudgeting_ios/blob/fd43c30d9ee7d075625997a607bdb1dd002cf1a8/Aspire Budgeting/Facilities/StateManager.swift#L159"></span>Lines should not have trailing whitespace.<br />
<code>trailing_whitespace</code> <code>StateManager.swift:159</code></td>
    </tr>
  </tbody>
</table>